ui/components/multichain/notifications-tag-counter/index.scss

```diff
@@ -0,0 +1,11 @@
+.notifications-tag-counter {
+  padding-left: 6px;
+  padding-right: 6px;
+}
+
+.notifications-tag-counter__unread-dot {
+  font-size: 10px;
+  padding-left: 3px;
+  padding-right: 3px;
+  line-height: 1.4;
+}
```

Comment:
This CSS file structure should get changed to:

```
.notifications-tag-counter {
  padding-left: 6px;
  padding-right: 6px;

  &__unread-dot {
    font-size: 10px;
    padding-left: 3px;
    padding-right: 3px;
    line-height: 1.4;
  }
}
```

Reply:
thx!

Reply:
We follow strict BEM?